README.MD

```diff
@@ -88,6 +88,28 @@ Presidio leverages:
 * [GRPC](https://grpc.io)
 * [re2](https://github.com/google/re2)
 
+## Give it a try - Run locally with KIND
+Presidio is built for Kubernetes, however you can give it a try using [KIND (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind).
+
+1. Install [Docker](https://docs.docker.com/install/).
+2. Clone Presidio.
+3. Run
+
+    ```sh
+    cd deployment/
+    ./run-with-kind.sh
```

Comment:
Some issues I had with installing on an ubuntu VM:
1. ./run-with-kind.sh required sudo
2. After it finishes, it says "presidio is now installed! To find out about your newly configured system, run: 
`$ help status presidio-demo`
However running this (with sudo) returns:
`Error: release: not found`
3. Port forwarding required sudo in both commands:
`sudo kubectl port-forward -n presidio $(sudo kubectl get pod -n presidio -l app=presidio-demo-presidio-api -o jsonpath='{.items[0].metadata.name}') 8080:8080`
Worth adding an & at the end as it blocks the terminal window

Reply:
After completing this, all worked well. Pretty cool!
Maybe we should have the prerequisites scripted in prerequisites.sh (and say that if you don't need to install a b c you can skip it). The other script should install kind and presidio (and maybe do port forwarding?)
I agree we shouldn't assume that users have kind installed.

Reply:
I will add prereqs script

Reply:
If Kind is already cloned it will re-clone it.
Perhaps first check if it's already there.
Running the script twice in a row will cause:
1. Cloning again KIND.
2. Create a namepsace again. It will fail because it exists.
3. ./deploy-helm.sh Must run only once. It will fail with erros.
4. ./deploy-helm.sh Is obsolete because Helm 3 does not use Tiller. I got an error there.

If there are assumptions about running this script only once - it should be stated, or change the script to allow it running multiple times

Reply:
added comment about the one time.
i dont want to bloat this script with retry mechanism and cleanups. sounds overkill